cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -37,6 +37,42 @@ components:
       - project_name
       - publication_doi
     keys:
+      organism_ontology_term_id:
+        type: curie
+        curie_constraints:
+          ontologies:
+            - NCBITaxon
+          allowed:
+            terms:
+              NCBITaxon:
+                - NCBITaxon:9606
+                - NCBITaxon:10090
+                - NCBITaxon:2697049
+                - NCBITaxon:32630
+                - NCBITaxon:7227
+                - NCBITaxon:7955
+                - NCBITaxon:6239
+                - NCBITaxon:9541
+                - NCBITaxon:9986
+                - NCBITaxon:9483
+                - NCBITaxon:9595
+                - NCBITaxon:9544
+                - NCBITaxon:9598
+                - NCBITaxon:9823
+                - NCBITaxon:30608
+                - NCBITaxon:10116
+            ancestors:
+              NCBITaxon:
+                - NCBITaxon:9541
+                - NCBITaxon:9544
+                - NCBITaxon:10090
+                - NCBITaxon:9986
+                - NCBITaxon:9598
+                - NCBITaxon:10116
+                - NCBITaxon:9823
+        add_labels:
+          - type: curie
+            to_column: organism
```

Comment:
i just copied and pasted this from `obs` to `uns` in the schema def. the `to_column` keyword doesn't make as much sense since this is a key not a column. i think it's slightly better to just keep the keywords consistent but i'm open to changing it

Reply:
I slightly prefer renaming it to `to_key` (or if you can think of a neutral term that captures both), but could be convinced otherwise

Reply:
i can rename it!